src/bin/mosaic-start.ts

```diff
@@ -45,9 +46,17 @@ mosaic
       // reuse params from node start command
       graphDescription.mosaicDir = mosaicDir;
       graphDescription.ethereumRpcPort = rpcPort;
-      const graph = new Graph(graphDescription);
-      graph.start();
-    }
 
+      let graphStart;
+
+      // options.origin passed only in case of starting an auxiliary chain
+      if (options.origin) {
+        graphStart = new GraphStart(graphDescription, options.origin, chain);
+      } else {
+        graphStart = new GraphStart(graphDescription, chain, null);
```

Comment:
This seems little confusing. 
constructor of GraphStart expects 
```
  public constructor(graphDescription: GraphDescription, originChain: string, auxiliaryChain: string) {

```
Second argument is originChain and third is aux chain. But someone starts 1407, the auxilary chain is passed at the place of origin chain?

Reply:
For starting 1407 (auxiliaryChain), we pass `options.origin` as `ropsten`. 
`graphStart = new GraphStart(graphDescription, ropsten, 1407);`